cmd/dep/vndr_importer.go

```diff
@@ -123,7 +124,10 @@ func (v *vndrImporter) convert(pr gps.ProjectRoot) (*dep.Manifest, *dep.Lock, er
 		fb.NewLockedProjectFeedback(lp, fb.DepTypeImported).LogFeedback(v.logger)
 	}
 
-	return manifest, lock, nil
+	if len(manifest.Constraints) > 0 {
```

Comment:
Sorry, dayjob on fire all day yesterday and most of this morning.  Looking again now..